examples/api/test/widgets/table/table.0_test.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/table/table.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Table example has correct view on screen', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.TableExampleApp());
+
+    // Check if appbar is visible with given title
+    expect(find.widgetWithText(AppBar, 'Table Sample'), findsOneWidget);
+
+    // Check if table is visible
+    expect(find.byType(Table), findsOneWidget);
+    final Table table = tester.widget<Table>(find.byType(Table));
```

Comment:
```suggestion
    final Table table = tester.widget<Table>(find.byType(Table));
```

If we're getting `Table` widget, other line becomes redundant.

Reply:
Ah, I thought that would throw an exception but now I realize if it is throwing an exception then it is a failed test. 😀